ocaml-lsp-server/src/ocaml_lsp_server.ml

```diff
@@ -663,12 +604,16 @@ let on_request :
   | TextDocumentHighlight req -> later highlight req
   | DocumentSymbol { textDocument = { uri }; _ } -> later document_symbol uri
   | TextDocumentDeclaration { textDocument = { uri }; position } ->
-    later (fun state () -> definition_query `Declaration state uri position) ()
+    later
+      (fun state () -> Definition_query.run `Declaration state uri position)
```

Comment:
`Go_to_def.query` seems a bit more self-explanatory than `Definition_query.run` but not a strong opinion

Reply:
I think I prefer to just follow the names of lsp requests themselves.